src/Systems/BehaviourScriptSystem.cpp

```diff
@@ -19,7 +19,8 @@ BehaviourScriptSystem::~BehaviourScriptSystem() {
 void BehaviourScriptSystem::update(milliseconds deltaTime) {
     auto behaviourScripts = BehaviourScriptStore::getInstance().getAllBehaviourScripts();
     for (auto script: behaviourScripts) {
-        script.get().onUpdate(deltaTime);
+        if (script.get().isActive || EntityManager::getInstance().isEntityActive(script.get().entityId))
```

Comment:
Je wilt dit eigenlijk hier niet checken, maar bij het ophalen al. In ieder geval of het entiteit actief is